integration/options.go

```diff
@@ -41,7 +41,7 @@ const (
 	defaultUseTChannelClientForReading = true
 
 	// defaultUseTChannelClientForWriting determines whether we use the tchannel client for writing by default.
-	defaultUseTChannelClientForWriting = false
+	defaultUseTChannelClientForWriting = true
```

Comment:
lol meant to set `defaultUseTChannelClientForReading` to false but ended up flipping the wrong flag, will fix